src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -337,6 +337,82 @@ .maxstack  7
         }
 
         [Fact]
+        public void SimpleTupleNew2()
+        {
+            var source = @"
+class C
+{
+    static void Main()
+    {
+        var x = new (int a, int b)(1, 2);
```

Comment:
Would it also be useful to test constructor initializer? `var x = new (int a, int b)(1, 2) { a = 3; b = 4 };`